Spigot-API-Patches/0193-final-Brand-support-PaperAPI.patch

```diff
@@ -0,0 +1,30 @@
+From a532a0c1f3438190efe6d95184026dfe79cae736 Mon Sep 17 00:00:00 2001
+From: DigitalRegent <misterwener@gmail.com>
+Date: Mon, 6 Apr 2020 19:53:29 +0200
+Subject: [PATCH] final Brand support (PaperAPI)
+
+
+diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
+index 877ea4f3..b2352a8e 100644
+--- a/src/main/java/org/bukkit/entity/Player.java
++++ b/src/main/java/org/bukkit/entity/Player.java
+@@ -2063,6 +2063,16 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+         }
+     }
+ 
++    // paper start - brand support
```

Comment:
Paper comments are generally uppercase since it's a proper noun (So like `// Paper Start - brand support`)

Additionally, start and end comments usually both have their explanation next to them.

Do this will all your Paper comments